apps/web/app/(use-page-wrapper)/(main-nav)/event-types/page.tsx

```diff
@@ -19,14 +20,23 @@ export const generateMetadata = async () =>
 
 const Page = async ({ params, searchParams }: PageProps) => {
   const context = buildLegacyCtx(headers(), cookies(), params, searchParams);
-  const session = await getServerSessionForAppDir();
```

Comment:
this makes initial RSC fetch so slow, check https://app.campsite.com/cal/posts/58hm2drix99s